tools/test-tools/CHANGELOG.md

```diff
@@ -2,10 +2,10 @@
 
 ## vNext
 
-Dependency updates.
-
 ## 2.0.0
 
 ### ⚠ BREAKING CHANGES
 
-Update `typescript` dependency from `4.x` to `5.x`.
+- Dependency updates.
+- Update `typescript` dependency from `4.x` to `5.x`.
+- Initial port for package-to-test-port mapping is now 9000 instead of 8081.
```

Comment:
I don't think so, but I'd say a visible change in behavior like this would be breaking? If a theoretical consumer was using the new range for something else already, this would break them.

Reply:
Fair point. In which case we should probably document what the ranges we will use are, to make all of this more explicit.

Reply:
The README now states port 9000 is the default initial port. The full range depends on the number of packages in the workspace from which the tool is invoked; I added a note about that too.